### PR TITLE
GSet support / RIAK-2907

### PR DESCRIPTION
**Pull request by lukebakken**
Depends on basho/riak-php-client#145

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -41,7 +41,7 @@
     }
   ],
   "require": {
-    "basho/riak": "^3.3",
+    "basho/riak": "dev-features/lrb/gset-crdt-RIAK-2907",
```
Comment (alexmoore) [inline composer.json]: @christophermancini How do you track the moving dependencies as these move towards release?

**Review by alexmoore — APPROVED** [review]
+1

File: composer.json (hunk 1 above)
Comment (lukebakken) [reply composer.json]: We have to tag `riak-php-client` first, then update this line, then we can merge this PR. I'm pretty sure that's it...

Comment (christophermancini) [reply composer.json]: Yea, thats all I do, always main client first.

**Pull request merged**